docs/error-codes.md

```diff
@@ -1427,3 +1427,15 @@ This is technically possible if a custom assembly defines `DynamicDependencyAttr
     public override void TestMethod<[DynamicallyAccessedMembers(DynamicallyAccessedMemberTypes.PublicFields)] T>() {}
   }
   ```
+
+#### `IL2096`: Trim analysis: Reflection call 'reflection call' inside 'calling method' is trying to make a case insensitive lookup which is not supported by ILLink"
```

Comment:
```suggestion
#### `IL2096`: Trim analysis: Call to 'Type.GetType method' can perform case insensitive lookup of the type, currently ILLink can not guarantee presence of all the matching types"
```

There's no need to include the calling method, warnings always have location (origin) which is the calling method (or even more precisely the instruction).